src/com/sun/ts/tests/websocket/api/jakarta/websocket/clientendpointconfig/WSClient.java

```diff
@@ -0,0 +1,1158 @@
+/*
```

Comment:
Odd that github shows this file change differently than looking at it on my local machine via gitg tool.  What I see in github, after clicking "show diff" is what looks like a newly created file src/com/sun/ts/tests/websocket/api/jakarta/websocket/clientendpointconfig/WSClient.java, however, in the local gitg view, I see that we renamed the *javax*.java file to *jakarta*.java and only changed two lines (package name + change reference from javax class to com.sun.ts.tests.websocket.api.jakarta.websocket.clientendpointconfig.TCKConfigurator.class).

Reply:
no change is requested, just sharing the poor experience in github, in case others try to review this pr on github.

Reply:
When I first reviewed the PR in gh, I thought perhaps I had done something wrong, but as you said, it just seems to be the way git handles diffs on files that have been renamed AND changed.  For non-java files like build.xml where there was no package statement to change after `git mv`ing, the diff view correctly shows them as "File renamed without changes."  That wording of that label might lead one to think that "File renamed with changes." is possible, I wonder if this behavior is intentional, or a bug...

Reply:
Not sure if we will get an answer, but I tried tweeting about this with a link here via https://twitter.com/scottmarlow/status/1250411983498809344.

Reply:
I also see the same output with the "gitk" tool locally, as well as "git show commitid".  I'm starting to question whether the "gitg" tool is showing correct output.

Reply:
@frowe I think that there is something wrong with the pr, as I generated a patch (git format-patch -1 4320492ed) and see the same changes as we see in github.  I cannot explain why it is but it looks like changes are needed.

Reply:
I'm not sure what to do differently.  I did:
git mv on any paths/files with with javax
created an eclipse project for websocket
imported it
did a search and replace on javax
then a git commit -a -s
then a git push
I do see from https://stackoverflow.com/questions/433111/how-to-make-git-mark-a-deleted-and-a-new-file-as-a-file-move that mv'ing of directories can be problematic.  I could try the suggestion to split the mv and modify into two PR's.  Looking for advice, my gitfoo has run out...

Reply:
I'm not sure yet, [I found another issue](https://github.com/libgit2/libgit2/issues/4237) which gave me some ideas, not that it sounds like the exact same issue.

Reply:
I tried various different "git show -- COMMITID" options, non of them helped.  Perhaps there is a fix, I'm not sure.

I also tried "git show --summary 4320492ed" which generated "create mode 100644 src/com/sun/ts/tests/websocket/api/jakarta/websocket/clientendpointconfig/WSClient.java" in the output but also mentioned:

> warning: inexact rename detection was skipped due to too many files.
> warning: you may want to set your diff.renameLimit variable to at least 707 and retry the command.
> 

I looked up the diff.renameLimit and apparently, if I update my local .git/config file in my tck root folder, and add the following, I can see that the pr is correct with regard to the strangeness I saw on github and locally:
```
[diff]
        renameLimit = 6000
```

So, we should be able to make progress reviewing the changes now. :)